src/managers/GuildMemberManager.js

```diff
@@ -432,6 +433,40 @@ class GuildMemberManager extends CachedManager {
     return this._add(data, cache);
   }
 
+  /**
+   * Adds a role to a member.
+   * @param {GuildMemberResolvable} user
+   * @param {RoleResolvable} role
+   * @param {string} [reason]
+   * @returns {Promise<GuildMember|User|Snowflake>}
+   */
+
```

Comment:
There should not be an empty line between the jsdoc of a method and the method declaration.